app/domain/authentication/authn_jwt/identity_providers/create_identity_provider.rb

```diff
@@ -61,11 +63,24 @@ def validate_identity_configuration
         end
 
         def multiple_identities_configured
-          identity_from_decoded_token_provider.identity_available? && identity_from_url_provider.identity_available?
+          identity_from_decoded_token_available? && identity_from_url_available?
         end
 
         def no_identities_configured
-          !identity_from_decoded_token_provider.identity_available? && !identity_from_url_provider.identity_available?
+          !identity_from_decoded_token_available? && !identity_from_url_available?
+        end
+
+        def identity_from_decoded_token_available?
```

Comment:
You does not operate decoded token in the method so why it's in the method name?

Reply:
Its with question mark in end so it means its boolean.
Function return boolean value about availiability of identity in decoded jwt token

Reply:
`token-app-propery-is-available?`

Reply:
sounds good